uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/htmleditor/HtmlEditorView.java

```diff
@@ -231,19 +229,30 @@ private void handleColorSelection(MouseEvent event, Button button, Input input)
     @Override
     public void load() {
         if (!loaded) {
-            final String identifier = String.valueOf(System.currentTimeMillis());
-            final String editorId = "html-editor-" + identifier;
-            final String toolbarId = "html-editor-toolbar-" + identifier;
-
-            configureScreenComponents(editorId,
-                                      toolbarId);
-            loadEditor(editorId,
-                       toolbarId);
 
+            identifier = String.valueOf(System.currentTimeMillis());
             loaded = true;
+
+            configureScreenComponents(getEditorId(), getToolbarId());
+            loadEditor(getEditorId(), getToolbarId());
         }
     }
 
+    private String getToolbarId() {
```

Comment:
@karreiro thanks for the change :) Could you please also remove these helper methods? Now that destroyEditor no longer uses them, they're only used in `load()` method. I think it would be best to switch back to previous implementation which was using final string variables `identifier`, `editorId` and `toolbarId` inside `load()` method, minimizing their scope and changes introduced as well..

Reply:
@jhrcek Thanks for the suggestion. But I prefer to keep it as is in this case. Separating the "id construction logic" from the "load logic" is a good approach. Besides, (considering the open/closed principle) it’s easier to reuse the identifier in future cases.

Reply:
Ok let's leave it as is. FYI still waiting for full downstream build to verify locally..

Reply:
@jhrcek I think that the full downstream passes. But, I did a new commit removing the destroyEditor parameters.